### PR TITLE
fix admin modal rendering

### PR DESCRIPTION
**Pull request by FinalAngel**
Related issue for cleanup: https://github.com/aldryn/aldryn-bootstrap3/issues/28

- modal fixes for image
- modal fixes for button
- modal fixes for label

should be like:
image

instead is:
image

### CODE REVIEW

**Comment by Marketionist** [comment]
Looks like the same issue occurs while adding Link/Button plugin

**Comment by FinalAngel** [comment]
alright I consolidate here

File: aldryn_bootstrap3/templates/admin/aldryn_bootstrap3/plugins/image/change_form.html [hunk 1]
```diff
@@ -128,15 +122,13 @@
...
-
-
     </style>
 
     {{ block.super }}
 
     {# Javascript for preview (must be after the rest of the fields, because we rely on global variables set there. #}
     <script>
-        if (!$) {
+        if (!window.$) {
```
Comment (vxsx) [inline aldryn_bootstrap3/templates/admin/aldryn_bootstrap3/plugins/image/change_form.html]: maybe like this 
```
(function ($) {
    $(document).ready(function () {
        ...
    })
}(window.jQuery || window.CMS.jQuery || window.django.jQuery));
```

**Comment by vxsx** [comment]
LGTM

File: aldryn_bootstrap3/templates/admin/aldryn_bootstrap3/plugins/image/change_form.html (hunk 1 above)
Comment (FinalAngel) [inline aldryn_bootstrap3/templates/admin/aldryn_bootstrap3/plugins/image/change_form.html]: yes there is a separate issue to refactor

**Pull request merged**